src/cmd/property.go

```diff
@@ -11,6 +11,73 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var examplePropertyCmd = &cobra.Command{
+	Use:   "property",
+	Short: "Example Property",
+	Long:  `Example Property`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(getExample[opslevel.PropertyInput]())
+	},
+}
+
+var getPropertyCmd = &cobra.Command{
+	Use:        "property",
+	Short:      "Get details about an assigned property",
+	Long:       `Get details about an assigned property`,
+	Example:    `opslevel get property owner-alias property-id`,
+	Args:       cobra.ExactArgs(2),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		ownerId := args[0]
+		propertyId := args[1]
+
+		result, err := getClientGQL().GetProperty(ownerId, propertyId)
+		cobra.CheckErr(err)
```

Comment:
```suggestion
		cobra.CheckErr(err)
		if result.Definition.Id == "" && result.Owner.Id() == "" {
			err = fmt.Errorf("property '%s' on '%s' not found\n", propertyId, ownerId)
			cobra.CheckErr(err)
		}
```

There should be a condition to check if the returned definition and owner. Otherwise an unassigned property results in this output (with exit code 0):

```
~/repos/cli/src ta/list-service-props ?1 ❯ o get property computer Z2lkOi8vb3BzbGV2ZWwvUHJvcGVydGllczo6RGVmaW5pdGlvbi8xODA                                                                                          Ruby 3.0.0 02:45:08 pm
{
  "Definition": {
    "id": null
  },
  "Owner": {
    "OnService": {
      "id": null
    }
  },
  "ValidationErrors": null,
  "Value": null
}

```